Add propertyNames validator to draft6

torce opened the pull request:
I have implemented `propertyNames` for draft6 validator.

After executing the `update_common_tests` I tested it against the common tests suite and the draft6/propertyNames tests are passing.

bastelfreak commented:
Hi @torce ,
thanks for the PR. The project was recently migrated to Vox Pupuli. Can you please rebase your branch and add tests for the change?

torce commented:
@bastelfreak
- Solved conflicts with master branch.
- I have added a simple test for `propertyNames`.
- Common test suite was failing since draft7 is not supported. I have added ignore rules for all draft7 tests.

bastelfreak commented:
thanks for the work!
